docker-entrypoint.sh

```diff
@@ -49,10 +49,28 @@ if [ ${NGINX_CORS_ENABLED} == 1 ]; then
   mkdir -p /etc/nginx/conf.d/fragments/location/php
   if [ ! -z ${NGINX_CORS_DOMAINS} ]; then
     print "Activating filtered CORS on domains: ${NGINX_CORS_DOMAINS}"
-    envsubst '${PHP_HOST} ${PHP_PORT} ${NGINX_DEFAULT_SERVER_PORT} ${NGINX_DEFAULT_SERVER_NAME} ${NGINX_DEFAULT_ROOT} ${DEFAULT_SERVER} ${NGINX_CORS_DOMAINS}' < /templates/fragments/location/php/cors-filtered.conf > /etc/nginx/conf.d/fragments/location/php/cors.conf
+    envsubst '${NGINX_CORS_DOMAINS}' < /templates/fragments/location/php/cors-filtered.conf > /etc/nginx/conf.d/fragments/location/php/cors.conf
   else
     print "Activating unfiltered CORS"
-    envsubst '${PHP_HOST} ${PHP_PORT} ${NGINX_DEFAULT_SERVER_PORT} ${NGINX_DEFAULT_SERVER_NAME} ${NGINX_DEFAULT_ROOT} ${DEFAULT_SERVER}' < /templates/fragments/location/php/cors-unfiltered.conf > /etc/nginx/conf.d/fragments/location/php/cors.conf
+    cp /templates/fragments/location/php/cors-unfiltered.conf /etc/nginx/conf.d/fragments/location/php/cors.conf
+  fi
+fi
+
+if [ ! -z ${NGINX_OSB_BUCKET} ]; then
```

Comment:
what happens here with projects (we have plenty of them) which requires a custom osb.conf file ? 
we are basically overwriting everything without giving a notice to the users.

Reply:
You are right, because even if we copy the file when we build the image on the single project, in fact the entrypoint is executed last and without a check if the file exists or not we would always overwrite it. I've added a missing check, that is we copy the file if the variable is not empty and if it doesn't already exist:

`if [ ! -z ${NGINX_OSB_BUCKET} -a ! -f "/etc/nginx/conf.d/fragments/osb.conf" ]; then`